### PR TITLE
Javascript rayhan

### PR DESCRIPTION
**Pull request by sansaid**

### CODE REVIEW

File: week-1/Homework/mandatory/1-syntax-errors.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
-
+function introduceMe(name, age){
+return "Hello, my name is " + name + "and I am " + age + "years old";
```
Comment (sansaid) [inline week-1/Homework/mandatory/1-syntax-errors.js]: Good job! Have you considered using string interpolation here instead to make your code neater? What about indentation here?

File: week-1/Homework/mandatory/3-function-output.js [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 function getNumber() {
-    return Math.random() * 10;
+    return Math.random() * 10;//returns a random number between 0 and 10
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
-    return w1.concat(w2);
+    return w1.concat(w2);//this adds w1 to w2 as string
 }
```
Comment (sansaid) [inline week-1/Homework/mandatory/3-function-output.js]: Good job with commenting!

File: week-1/Homework/mandatory/4-tax.js [hunk 3]
```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+    return (price) * 20 / 100;
+}
```
Comment (sansaid) [inline week-1/Homework/mandatory/4-tax.js]: Does this return the correct value? Have you tested this?

File: week-1/Homework/mandatory/5-magic-8-ball.js [hunk 4]
```diff
@@ -45,7 +45,9 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+function shakeBall() {
+
```
Comment (sansaid) [inline week-1/Homework/mandatory/5-magic-8-ball.js]: You haven't written any code here. Did you commit the latest version of your code?

Comment (Rayhan94) [reply week-1/Homework/mandatory/5-magic-8-ball.js]: Hi Sanyia,

Thank you for feedback its useful and I have prettier installed. I think the calculator tax returns correct.

I will do shakeball soon.

Rayhan

File: week-2/Homework/mandatory/1-fix-functions.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
+// The below functions are syntactically correct but not outputting the right results.
+// Look at the tests and see how you can fix them.
+
+function mood() { //creating a function called mood
+    let isHappy = true; //assigning boolean variables either true or false
+    let isHappy = false;
```
Comment (ChrisOwen101) [inline week-2/Homework/mandatory/1-fix-functions.js]: Have you tried running this code to see if it will work? I don't think this will compile

**Pull request closed without merging**